src/Plugin/HistoryPlugin.php

```diff
@@ -40,7 +39,7 @@ public function handleRequest(RequestInterface $request, callable $next, callabl
             $journal->addSuccess($request, $response);
 
             return $response;
-        }, function (Exception $exception) use ($request, $journal) {
+        }, function (\Exception $exception) use ($request, $journal) {
```

Comment:
You need to add a typecheck here and only add the exception to the journal if it is an HTTP exception.